solver/src/http_solver/model.rs

```diff
@@ -42,6 +43,12 @@ pub struct TokenInfoModel {
     pub decimals: u32,
 }
 
+#[derive(Debug, Serialize)]
+pub struct CostModel {
+    pub amount: u128,
```

Comment:
can the cost not be negative in case of EOA orders where the fee is larger than their own marginal execution cost (since e.g. the fee was estimated with a multi-hop baseline in mind). Thus including this order has a negative cost.

Reply:
There's a separate `fee` field to make things more explicit.

Reply:
Not sure I understand. This models the fixed cost of including the order (the gas cost). Whatever the fee we charge to our customers is, our gas cost is always a positive number right?

Reply:
I think initially, the idea was to add this cost field for Uniswaps, where it simply represents the gas cost. For orders, we also collect a fee, and we were considering to mix this fee into the order cost (thus making the cost negative). But later we decided to introduce a separate `fee` field for orders, so that `costs` indeed only represents the marginal execution cost.

Reply:
Right, the fee in the AMM model is the fee the AMM charges to us, while the fee in the orders is the fee we charge our customers. They are both proportional, and they both affect the optimization. I don't mind call it differently though, do you have any suggestion?

Reply:
If I'm the only one confused by this I guess it's fine to keep it like this. Maybe `protocolFee` would express that this is the fee for our protocol?